### ISSUE TITLE
Non-Entries/Comments are not interpreted correctly

### ISSUE THREAD

**Issue by nils-werner** [sciunto-org/python-bibtexparser#133]
According to [btxdoc](http://mirrors.ctan.org/biblio/bibtex/base/btxdoc.pdf), §4, item 7, p. 13

> BibTeX allows in the database files any comment that's not within an entry. If you want to comment out an entry, simply remove the `@` character preceding the entry type.

However when you parse the following database

```bibtex
@ARTICLE{bla,
    author={Fartsy},
    title={Title},
}

year=nonsense,
```

`year` gets added to the entry like

```python
{u'}\n\nyear': u'nonsense,', 'ID': 'bla', ... }
```

**Comment by nils-werner** [comment]
Additionally, biber allows the LaTeX comment symbol `%` to be used, e.g.

```bibtex
@ARTICLE{bla,
    author={Fartsy},
    %title={Title},
}
```

however bibtexparser simply ignores the character and creates a field `%title` from it.

**Comment by Phyks** [comment]
Original issue is solved thanks to #64.

Not sure whether adding `%` support is a good idea. What do you think @sciunto?

**Comment by sciunto** [comment]
I think we should probably support that better.
It means we have to consider 
```
@ARTICLE{bla,
    author={Fartsy},
    %   title={Title},
}

```
as well.

The minimum is to add a note to the documentation.
A partial support would be to make sure that the example above creates a field `%title` or ignore the field. Whatever the behaviour we choose, it should be documented.

**Comment by omangin** [comment]
Right now having something like `%title={Title},` or `%  title={Title},` raises an error on parsing.

Supporting the former is quite trivial by extending characters allowed in field names (replace [bibtexexpression.py#L145](https://github.com/sciunto-org/python-bibtexparser/blob/master/bibtexparser/bibtexexpression.py#L145) by `field_name = pp.Word(pp.alphanums + '_-()%')('FieldName')` but it does not cover the latter. We could alternatively easily enable commented fields if this is desired but what to do with the following?
```
abstract = {A multi-line value with a
            % in the
            middle of it},
```
```
%abstract = {A multi-line
             commented field},
```

**Comment by Phyks** [comment]
Yes, supporting comments can really quickly lead to such issues :/ I'll try to have a deeper look at what Biber exactly does.

**Comment by MiWeiss** [comment]
Relevant parts for v2 extracted to its own issue: #372
